config/environments/development.rb

```diff
@@ -51,8 +51,8 @@ module OptimizerWrapper
   OptimizerLogger.with_datetime = true
   OptimizerLogger.caller_location = :relative
 
-  I18n.available_locales = [:en, :fr]
-  I18n.default_locale = :en
+  I18n.available_locales = [:en, :fr, :unspecified]
```

Comment:
trouver autre chose si possible

Reply:
La seule alternative que je vois serait, à chaque fois que l'on fait appel à _I18n.t('export_file....)_ de vérifier qu'une langue est bien fournie et, sinon, mettre notre valeur par défaut. 

Cela revient à rajouter 26 if / else.

Reply:
@fab-girard since we want to keep the old column labels for compatibility reasons, we need to have a separate "language" which represents them. It looks like `:unspecified` is the cleanest way to do this. Otherwise, like @fonsecadeline said, it will be an if/else party which will be harder to manage in the long run.

Reply:
@fab-girard what should we do about this ?